policyengine_us/variables/gov/states/hi/tax/income/exemptions/hi_disabled_exemptions.py

```diff
@@ -40,4 +40,6 @@ def formula(tax_unit, period, parameters):
             disabled_exemption_non_disabled_spouse,
         )
 
-        return disabled_exemption_head + disabled_exemption_spouse
+        return (
+            disabled_exemption_head + disabled_exemption_spouse
+        ) * disabled_head
```

Comment:
they get zero if only the spouse is disabled?

Reply:
Sorry, the tax form instruction does not mention the case that only the spouse is disabled, so I wrongly think the spouse has to be the filer to claim the additional disabled exemption. However, the law does mention that case. I rewrote the formula according to state law and added some extra test units. Now, it should be correct for all the cases.